src/charts/tableView.js

```diff
@@ -12,14 +12,16 @@ define(['jquery', 'moment', 'lokijs', 'charts/chartingRequestMap', 'websockets/s
     table.animate({left: '0'}, 250);
     chart.animate({left: '-100%'}, 250);
     refresh_table(dialog, key); /* clear table and fill it again */
+    dialog.parent().css('width', 520);
```

Comment:
@aminroosta The size of table is not changing until the dialog width changes,so I'm using  this constant value that is needed to avoid horizontal scroll.